getting-started/configure-authentication.md

```diff
@@ -25,14 +25,14 @@ After registration, you should have a `client id` and `client secret` value for
 
 By default, a user who logs into SystemLink Enterprise will have minimal permissions and will not be able to make changes to the system. At installation, you must assign an initial administrator by defining a mapping for a specific Open ID Connect claim. The administrator will be able to log in, perform initial configuration of the application, and assign roles to other users.
 
-1. In systemlink-values.yaml, set `initialAdministratorMapping.mappingKey` to the claim to use. The default is "email".
-2. Set `initialAdministratorMapping.mappingValue` to the value that identifies the administrator.
+1. In systemlink-values.yaml, set `userservicesetup.initialAdministratorMapping.mappingKey` to the claim to use. The default is "email".
```

Comment:
Also, we are tracking changes for inclusion in the release notes here: https://nio365.sharepoint.com/:x:/r/sites/ASWSystemLink/Shared%20Documents/SLE%20Development/Helm%20Chart%20Breaking%20Changes%20for%20the%20May%202022%20Release.xlsx?d=w114e7ff1a3124c508b93f03a04712330&csf=1&web=1&e=kLXGu3